db/schema.rb

```diff
@@ -179,6 +187,26 @@
     t.index ["email"], name: "index_users_on_email", unique: true, using: :btree
   end
 
+  create_table "wind_turbine_speed_measurements", force: :cascade do |t|
```

Comment:
Tables added to schema.rb, but there is no migration for them. Where are those migrations?